components/notifcard/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FunctionComponent, useContext } from 'react';
+
+import { faTrash } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { Select, SelectOption, TextInput } from '../';
+import { NotificationContext } from '../../contexts/NotificationContext';
+import { notificationForms } from '../../utils/notificationForms';
+
+export interface NotifCardProps {
+  id: string;
+  type: string;
+}
+
+const NotifCard: FunctionComponent<NotifCardProps> = ({ id, type }) => {
+  const {
+    notificationData,
+    handleUpdateNotificationType,
+    handleUpdateNotificationData,
+    handleRemoveNotification,
+  } = useContext(NotificationContext);
+
+  return (
+    <div className="border border-solid rounded-md mb-8">
+      <div className="flex flex-row items-center justify-between p-4 bg-gray-50 border-b">
+        <p>Notification #{id}</p>
+        {notificationData.length < 0 && (
```

Comment:
Did you mean `notificationData.length > 0`?

Reply:
Ah, I mean `notificationData.length > 1`